src/config/dynamic-theme-fixes.config

```diff
@@ -460,6 +460,20 @@ INVERT
 
 ================================
 
+bfi.org.uk
+
+INVERT
+img[src$="bfi_logo_transp.png"]
+img[src$="national-lottery-logo-color.png"]
+img[src$="sight-and-sound-logo-280x69.png?itok=AEegRSbk"]
```

Comment:
`img[src*="sight-and-sound-logo-280x69.png"]`

Unsure what itok is but seems it can change

Reply:
Done.